### DISCUSSION TITLE
Can't remove DRM from ebooks.com book

### DISCUSSION THREAD

**Discussion by PFBourassa** [apprenticeharper/DeDRM_tools#2141] [Q&A]
I'm new to this, but I have hit a wall.
I can open the book using Overdrive on Windows 10, so it's not corrupt or anything.
My goal is to move the book to my Kindle Paperwhite.

When I try to remove the DRM, the log says:
[log.txt](https://github.com/apprenticeharper/DeDRM_tools/files/9595747/log.txt)

`DeDRM v7.2.1: Verifying zip archive integrity
DeDRM v7.2.1: “210443934.epub” is neither an Adobe Adept nor a Barnes & Noble encrypted ePub
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "calibre\customize\ui.py", line 175, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 644, in run
  File "calibre_plugins.dedrm.__init__", line 420, in ePubDecrypt
calibre_plugins.dedrm.DeDRMError: DeDRM v7.2.1: Couldn't decrypt after 0.1 seconds. DRM free perhaps?`

I know the book isn't DRM free because it won't open in Calibre for reading, and Calibre can't send it to my Kindle.
And the website asked me to install Adobe Digital Edition to download it. But Adobe Digital Edition just says I already used the key or something.
![DigitalEditions_Error](https://user-images.githubusercontent.com/881264/190938893-94a67781-1970-404c-a3cc-e1c3d50685ee.png)

What else can I try?

Thank you!

**Comment by ElleKayEm** [comment]
DeDRM works with Adobe DRM epubs/pdfs by using a key associated with your Adobe ID.  Since you initially used Overdrive instead of ADE, I'm not sure how or if you can decrypt it.
